myfirstgame.html

```diff
@@ -1,32 +1,45 @@
 <script>
-
 var guess, 
 	message;
-guess = prompt ("Guess how I got to school today?");
-
-switch (guess){
-	case "walked":
-	message = "I did walk today, you got it on your first try!"
-	break;
 
-	case "drove":
-	message = "I did not arrive on 4 wheels today, try again."
-	break;
+//Got loop to run, added alert messages, still struggling on getting loop to recognize changing guess variable.//
 
-	case "bused":
-	message = "I alomost took the bus but changed my mind, try again."
-	break;
-
-	case "biked":
-	message = "My bike has no brakes right now, try again."
-	break;
+guess = prompt ("Guess how I got to school today?");
 
-	case "ran":
-	message = "I get too sweaty if I run, try again."
-	break;
+switch (guess){
+		
+		case "walked":
+		alert(message) = "I did walk today, you got it on your first try!"
+		guess = prompt ("Guess how I got to school today?");
+		break;
+
+		case "drove":
+		message = "I did not arrive on 4 wheels today, try again.";
+		alert(message);
```

Comment:
the last two lines of each case are identical. repetition is usually a signal to simplify.